libcodechecker/cmd/cmd_line_client.py

```diff
@@ -85,6 +85,31 @@ def get_runs(client, run_names):
     return client.getRunData(run_filter)
 
 
+def valid_filter_value(user_values, valid_values):
+    """
+    Check if the value provided by the user is a valid value.
+    """
+    non_valid_values = []
+
+    for status in user_values.split(','):
+        ret = valid_values.get(status.upper(), None)
+        if ret is None:
+            non_valid_values.append(status)
+
+    if non_valid_values:
+        LOG.error('Invalid filter value:')
```

Comment:
Maybe it would be better to write the valid/invalid values in one line:
```py
invalid_values = ','.join(map(lambda x: x.lower(), non_valid_values))
LOG.error('Invalid filter value(s): {0}'.format(invalid_values))

valid_values = ','.join(map(lambda x: x.lower(), valid_values.keys()))
LOG.error('Valid filter values are: {0}'.format(valid_values))
```